### PR TITLE
Installation cleanup

### PR DESCRIPTION
**Pull request by jburel**
Clean up the installation script:
 
 * use the correct walkthrough
 * remove apache section

cc @hflynn

### CODE REVIEW

**Comment by hflynn** [comment]
Walkthroughs look good.

Apache is still listed on https://www.openmicroscopy.org/site/support/omero5.3-staging/sysadmins/unix/server-installation.html without mention of it being deprecated - do you want to edit on this PR since you have the other apache removals here?

**Comment by jburel** [comment]
@hflynn: done

File: omero/sysadmins/unix/server-installation.txt [hunk 1]
```diff
@@ -250,19 +250,18 @@ latest release is available from the :zeroc:`ZeroC website
...
 
-Please install one of :program:`apache` (and its wsgi module) or
-:program:`nginx` and Gunicorn in order to run OMERO.web.
+Please install :program:`nginx` and Gunicorn in order to run OMERO.web.
 
 +-----------+--------------------------------------+
 | System    | Packages                             |
 +===========+======================================+
-| BSD Ports | www/apache24 www/nginx               |
+| BSD Ports | www/nginx                            |
```
Comment (hflynn) [inline omero/sysadmins/unix/server-installation.txt:258]: Is this 'www/' here significant? Do we need to keep this table now?

Comment (jburel) [reply omero/sysadmins/unix/server-installation.txt:258]: this is the name of the package so yes we need it
I think the table should be kept

Comment (hflynn) [reply omero/sysadmins/unix/server-installation.txt:258]: Oh, np, just seemed worth checking.

**Comment by hflynn** [comment]
Looks good on staging, thanks, merging to allow further work on Ubuntu walkthroughs

**Pull request merged**